Assets/HoloToolkit/Input/Scripts/GamePad/XboxController/XboxControllerMapping.cs

```diff
@@ -84,67 +104,187 @@ public static void SetMapping(XboxControllerMappingTypes type, string value)
                     break;
                 case XboxControllerMappingTypes.XboxDpadVertical:
-                    XboxDpadVertical = string.IsNullOrEmpty(value) ? "XBOX_DPAD_VERTICAL" : value;
+                    XboxDpadVertical = string.IsNullOrEmpty(value) ? XBOX_DPAD_VERTICAL : value;
                     break;
                 case XboxControllerMappingTypes.XboxLeftTrigger:
-                    XboxLeftTrigger = string.IsNullOrEmpty(value) ? "CONTROLLER_LEFT_TRIGGER" : value;
+                    XboxLeftTrigger = string.IsNullOrEmpty(value) ? CONTROLLER_LEFT_TRIGGER : value;
                     break;
                 case XboxControllerMappingTypes.XboxRightTrigger:
-                    XboxRightTrigger = string.IsNullOrEmpty(value) ? "CONTROLLER_RIGHT_TRIGGER" : value;
+                    XboxRightTrigger = string.IsNullOrEmpty(value) ? CONTROLLER_RIGHT_TRIGGER : value;
                     break;
                 case XboxControllerMappingTypes.XboxSharedTrigger:
-                    XboxSharedTrigger = string.IsNullOrEmpty(value) ? "XBOX_SHARED_TRIGGER" : value;
+                    XboxSharedTrigger = string.IsNullOrEmpty(value) ? XBOX_SHARED_TRIGGER : value;
                     break;
                 case XboxControllerMappingTypes.XboxA:
-                    XboxA = string.IsNullOrEmpty(value) ? "XBOX_A" : value;
+                    XboxA = string.IsNullOrEmpty(value) ? XBOX_A : value;
                     break;
                 case XboxControllerMappingTypes.XboxB:
-                    XboxB = string.IsNullOrEmpty(value) ? "XBOX_B" : value;
+                    XboxB = string.IsNullOrEmpty(value) ? XBOX_B : value;
                     break;
                 case XboxControllerMappingTypes.XboxX:
-                    XboxX = string.IsNullOrEmpty(value) ? "XBOX_X" : value;
+                    XboxX = string.IsNullOrEmpty(value) ? XBOX_X : value;
                     break;
                 case XboxControllerMappingTypes.XboxY:
-                    XboxY = string.IsNullOrEmpty(value) ? "XBOX_Y" : value;
+                    XboxY = string.IsNullOrEmpty(value) ? XBOX_Y : value;
                     break;
                 case XboxControllerMappingTypes.XboxView:
-                    XboxView = string.IsNullOrEmpty(value) ? "CONTROLLER_LEFT_MENU" : value;
+                    XboxView = string.IsNullOrEmpty(value) ? CONTROLLER_LEFT_MENU : value;
                     break;
                 case XboxControllerMappingTypes.XboxMenu:
-                    XboxMenu = string.IsNullOrEmpty(value) ? "CONTROLLER_RIGHT_MENU" : value;
+                    XboxMenu = string.IsNullOrEmpty(value) ? CONTROLLER_RIGHT_MENU : value;
                     break;
                 case XboxControllerMappingTypes.XboxLeftBumper:
-                    XboxLeftBumper = string.IsNullOrEmpty(value) ? "XBOX_LEFT_BUMPER" : value;
+                    XboxLeftBumper = string.IsNullOrEmpty(value) ? CONTROLLER_LEFT_BUMPER_OR_GRIP : value;
```

Comment:
I'm getting an ArgumentException with this change, since it doesn't look like these axes were updated in InputManager.asset as well.

Reply:
Which also means they should be updated in the Add Input Manager Axes menu

Reply:
Good catch.

I was trying to keep in sync with your axis manager, but these must have gotten past me.